### DISCUSSION TITLE
Doesn't look like acorn-io/acorn is a known repository (but it is)

### DISCUSSION THREAD

**Discussion by liorsbg** [community/community#31105] [Other Feature Feedback, Questions, & Ideas]
This is a real repo: https://github.com/acorn-io/acorn/
But https://cs.github.com/acorn-io/acorn/ shows:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/13170066/187064747-62c6f2e1-0426-4e28-aadd-6dcc05aef003.png">

**Comment by SaptarshiSarkar12** [comment]
Check it now, it shows in my browser.

![Screenshot (6)](https://user-images.githubusercontent.com/105960032/187066156-dd31039c-2f4e-4d59-9ae3-d119902a24bd.png)

![image](https://user-images.githubusercontent.com/105960032/187065802-e9872d97-9eff-4daf-a768-a8708e560ea7.png)
